tests/PhpSpreadsheetTests/Worksheet/DrawingTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace PhpOffice\PhpSpreadsheetTests\Worksheet;
+
+use PhpOffice\PhpSpreadsheet\Reader\Xls;
+use PHPUnit\Framework\TestCase;
+
+class DrawingTest extends TestCase
+{
+    public function testCloningWorksheetWithImages()
+    {
+        $filename = './data/Reader/Xls/with_images.xls';
+        $reader = new Xls();
+        $spreadsheet = $reader->load($filename);
+        $worksheet = $spreadsheet->getSheetByName('Sheet1');
+        $originDrawingCount = count($worksheet->getDrawingCollection());
+
+        $clonedWorksheet = clone $worksheet;
+        $clonedDrawingCount = count($clonedWorksheet->getDrawingCollection());
+        self::assertEquals($originDrawingCount, $clonedDrawingCount);
+    }
```

Comment:
You should assert that the drawing collection is not the exact same object and was indeed cloned. Probably with `assertNotSame`.

 And same for worksheet, it must be cloned. So the patch might be too simple as it is now.